### PR TITLE
Add clip term to glossary

### PR DESCRIPTION
**Pull request by patcon**
Re-ticketed from https://github.com/hyphacoop/handbook/issues/4

This term was very unfamiliar to me at first, and feels like a term-of-art that any members or others working for us should know.

(I saw your comment about it being finance-specific @benhylau, but respectfully disagree and think it feels right to include here while closing out the glossary ticket.)

### CODE REVIEW

**Comment by benhylau** [comment]
LGTM @dcwalk will make some language adjustments then merge.

File: glossary.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ AKA the thing that no one ever reads. Below we've provided tweet-sized definitio
 
 - [**Capitalism**](https://en.wikipedia.org/wiki/Capitalism) an economic system based on private ownership of the means of production with the goal of generating profits. See: [The Black Socialists of America Glossary of Socialist Terms](https://blacksocialists.us/mumbo-jumbo)
 - [**Co-operative**](https://www.ica.coop/en/cooperatives/what-is-a-cooperative): a people-centred enterprise owned, controlled, and run by and for members to realise common economic, social, and cultural needs and aspirations.
+- **Clip.** The percentage Hypha skims from costs billed to client, to cover its operating costs. Example: "We operate with a 25% clip on our billed rate."
```
Comment (dcwalk) [inline glossary.md]:
```suggestion
- **Clip**: the percentage Hypha takes out of costs invoiced to clients in order to cover operating costs (e.g., "we clip 25% of our billable project work to fund administrative costs and collective programs"). See: [Calculating Member Wages](https://handbook.hypha.coop/finance.html#calculating-member-wages)
```

Bringing in line with language on finance page

Comment (dcwalk) [reply glossary.md]: ping @patcon

Comment (patcon) [reply glossary.md]: Thanks! Sounds good, but noting that this changed example from a noun to verb. "the clip" is how root systems spoke of it, and i think it's how we do, no? "the clip" or "a 25% clip" etc

Comment (dcwalk) [reply glossary.md]: That example combines the language from the first and last sentence of the linked page, which is the only place I've seen it "officially" used:
> Each Member's pay from billable project work will have a percent-based portion taken from it to fund administrative costs and collective programs. ... The amount is calculated based on the invoiced amount before transaction costs. This clip, currently set at 25%, is the same across all projects during a period and can be revised as necessary by consensus.

In meetings we've used it both as a noun and verb to my recollection, and I don't seen an issue using with both noun and verb forms. If you want to keep it only as a noun, I've modified two words to make it so

Comment (benhylau) [reply glossary.md]: There is a PR that changes the referenced part, which I just merged. So here is new proposal:

```suggestion
- **Clip** a percent-based portion deducted from billable project work to fund administrative costs and collective programs of the Co-operative. See: [Calculating Member Wages](https://handbook.hypha.coop/finance.html#calculating-member-wages)
```

Comment (dcwalk) [reply glossary.md]: I've hidden my suggestion. I'm happy to go ahead with @benhylau 's updated suggestion

**Comment by benhylau** [comment]
Seems to incorporate what everyone wants and no push back for 3 days. Merge.

**Pull request merged**